app/assets/javascripts/components/users/users_page.jsx

```diff
@@ -35,10 +30,33 @@ class UsersPage extends Component {
     };
   }
 
+  // --------------------------------------------------
+  // Lifecycle
+  // --------------------------------------------------
+
+  componentWillMount() {
+    this.setState(UsersStore.getState());
+  }
+
+  componentDidMount() {
+    this._listener = UsersStore.listen((state) => this.setState(state));
```

Comment:
Check out some of the other component's that call `Store.listen`. The convention is not as you've done it anymore - update it and make sure you update the constructor initialization of `this._listener` as well!